src/app/core/services/issue.service.ts

```diff
@@ -154,16 +151,29 @@ export class IssueService {
     );
   }
 
-  private createAndSaveIssueModel(githubIssue: GithubIssue): boolean {
-    const issue = this.createIssueModel(githubIssue);
-    this.updateLocalStore(issue);
-    return true;
+  private createAndSaveIssueModels(githubIssues: GithubIssue[], shouldEmit: boolean = true): Issue[] {
```

Comment:
As mentioned in the comment regarding `updateLocalStore`, let's look at removing the shouldEmit flag.